### PR TITLE
translate model_sharing.md and llm_tutorial.md to chinese

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu

Hi,

Here is last two files of Tutorials section. And I have a problem to translate `moving parts`in llm_tutorial.md.
I think `moving parts` means "there are many things that must all happen for success. But only one thing must go wrong for failure."  And it's very hard to be translated as it is not direct by words.  Do you think it's reasonable to translate it to "components are complex and closely related "

Best

File: docs/source/zh/llm_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+>>> tokenizer = AutoTokenizer.from_pretrained("mistralai/Mistral-7B-v0.1")
+>>> tokenizer.pad_token = tokenizer.eos_token  # Most LLMs don't have a pad token by default
+>>> model = AutoModelForCausalLM.from_pretrained(
+...     "mistralai/Mistral-7B-v0.1", device_map="auto", load_in_4bit=True
+... )
+```
+
+### 生成的输出太短/太长
+
+如果在[`~generation.GenerationConfig`]文件中没有指定，`generate`默认返回20个tokens。我们强烈建议在您的`generate`调用中手动设置`max_new_tokens`以控制它可以返回的最大新tokens数量。请注意，LLMs（更准确地说，仅[解码器模型]）也将输入提示作为输出的一部分返回。
```
Comment (stevhliu) [inline docs/source/zh/llm_tutorial.md:147]: I think you're missing the link to the decoder-only models [chapter](https://huggingface.co/learn/nlp-course/chapter1/6?fw=pt) from the NLP course

**Review by stevhliu — COMMENTED** [review]
Great job finishing off the Tutorial section! 🥳 

Yeah, in this context, I think "moving parts" refers to the many different facets of using a LLM, such as generation strategies, prompt design, and optimization. So maybe not so much that the components of a LLM are complex and closely related (though that's also true) but that there are many ways to use a LLM.

**Comment by jiaqiw09** [comment]
@stevhliu hi, I just fix the review and keep "moving parts" translation.

Best

**Review by stevhliu — APPROVED** [review]
Thanks again! 🚀

**Pull request merged**